components/Footer.js

```diff
@@ -0,0 +1,68 @@
+export default function Footer({ footer }) {
+  const logo = footer[6].fields.logo.fields.file.url;
+  const nav = footer[6].fields.footerNavLinks;
+  const navItem = [];
+  const copy = footer[6].fields.closedCaption;
+  const { title } = footer[6].fields;
+  const description = footer[6].fields.companyDescription;
+  const phone = footer[6].fields.contactNumber;
+  const icons = footer[6].fields.socialNetworksSection;
+  const iconsItem = [];
+  icons.map((i) => iconsItem.push(
```

Comment:
Map function returns an array, so you don't have to push items individually into iconsItem
```suggestion
 iconsItem = icons.map(i =>
```